source/common/quic/envoy_quic_server_stream.cc

```diff
@@ -41,11 +41,6 @@ EnvoyQuicServerStream::EnvoyQuicServerStream(
 
   stats_gatherer_ = new QuicStatsGatherer(&filterManagerConnection()->dispatcher().timeSource());
   set_ack_listener(stats_gatherer_);
-  // TODO(https://github.com/envoyproxy/envoy/issues/23564): Remove this line when the QUICHE is
-  // updated with a more reasonable default expiry time for QUIC Datagrams.
-  if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.enable_connect_udp_support")) {
```

Comment:
Also remove its definition?

Reply:
Are you deprecating the runtime guard? If so, you might want to remove its definition `RUNTIME_GUARD(bool, envoy_reloadable_features_enable_connect_udp_support, ...)` in runtime_features.cc.

And also it seems that you are replacing the existing runtime guard with a quic flag. It may worth to add a release note about how to turn the feature off.

Reply:
I removed this block since `SetMaxDatagramTimeInQueue` is no longer necessary if `SetForceFlushForDefaultQueue` is set to true. I am not deprecating the runtime guard yet.

I think I should have put the runtime guard around `SetForceFlushForDefaultQueue` though. I will do it in a following commit.

Reply:
Please false deprecate it and use a new runtime guard for the new change.

Reply:
Why should we deprecate the `envoy.reloadable_features.enable_connect_udp_support` in this PR? It's still used throughout the code in the other places and the change in this PR is still for the CONNECT-UDP feature.

Reply:
Then please add a new runtime to guard the new behavior in this change.

Reply:
The change in this PR is guarded by `envoy.reloadable_features.enable_connect_udp_support` already. This PR is a bug fix work of the previous CONNECT-UDP support PR, so I don't think we should introduce a new runtime guard? Let me know if you still think we should introduce a new runtime guard.

Reply:
Oh, I didn't notice that SetForceFlushForDefaultQueue() is already guarded.